src/components/Navbar/styles.module.scss

```diff
@@ -3,6 +3,10 @@
   justify-content: space-between;
 }
 
+.brand a {
+  color: #fff;
```

Comment:
Hmm, it would be nice to reference color variables in the theme. I'm not worried about it, though. It's just a thought I had.